docs/source/markdown/podman-create.1.md

```diff
@@ -691,6 +691,11 @@ Tune the host's OOM preferences for containers (accepts -1000 to 1000)
 #### **--os**=*OS*
 Override the OS, defaults to hosts, of the image to be pulled. For example, `windows`.
 
+#### **--personality**=*persona*
+
+Personality sets the execution domain via linux personality.
```

Comment:
Please also point to a man page where users can read more on personalities.

Reply:
```suggestion
Personality sets the execution domain via Linux personality.
```